app/index.html

```diff
@@ -58,18 +61,18 @@
       <md-content ng-cloak flex id="map">
         <apt-map ng-style='{"width": "100%", "height": "100%"}'></apt-map>
         <md-button class="md-icon-button md-raised md-hue-2 event-list-button" hide-gt-sm ng-click="ec.toggleEventList()" aria-label='Open Event List'>
-          <!--<md-icon md-svg-icon="img/icons/menu.svg" ></md-icon>-->
+          <!--<md-icon md-svg-icon="img/iconsis/menu.svg" ></md-icon>-->
```

Comment:
Hmm... There's no such directory as `img/iconsis/`
When we switch back to SVG icons, this would introduce a bug.